avaframe/com5SnowSlide/com5SnowSlideCfg.ini

```diff
@@ -56,16 +57,16 @@ snowSlide = 1
 cohesionMaxStrain = 0.2
 # threshold to avoid dividing by 0 (similar to minRKern). So we disregard too short bonds
 minDistCohesion = 1.0e-3
-# cohesive surface tension used as elasticity modulus in N/m²
+# cohesive surface tension used as elasticity modulus [N/m²]
 cohesiveSurfaceTension = 50000
 
 
 #++++++++++++ Resistance force parameters
-# height of the obstacles
+# height of the obstacles [m]
 hRes = 100
-# resistance coeff
+# resistance coefficient
 cw = 1000
-# diameter of obstacles
+# diameter of obstacles [m]
 dRes = 10
-# spacing between obstacles
+# spacing between obstacles [m]
```

Comment:
Oh, this is a good point, this is not valid anymore! 
We collapsed these parameters to 1. Please take the values from com1DFACfg.ini and set really high values.

Reply:
Okay, should a description of the new parameter then be added here as well? Is this then just a resistance coefficient?

Reply:
Yes, copy the info from com1DFACfg.ini. And yes, it is a coefficient, so no units needed

Reply:
related issue #1025